addons/gut/input_sender.gd

```diff
@@ -375,4 +375,4 @@ func get_auto_flush_input():
 
 
 func set_auto_flush_input(val):
-	_auto_flush_input = val
\ No newline at end of file
+	_auto_flush_input = val
```

Comment:
Is this like a whitespace change or something? very curious how you managed to open this file and change it.

Reply:
sometimes godot just does stuff idk. i didn't intentionally change anything here